integration_test/pg/storage_test.exs

```diff
@@ -33,19 +27,10 @@ defmodule Ecto.Integration.StorageTest do
     assert Postgres.storage_up(correct_params) == {:error, :already_up}
   end
 
-  test "storage up (wrong credentials)" do
-    refute Postgres.storage_up(wrong_user) == :ok
```

Comment:
To do this, you may need to start the connection and run the query inside a separate process (it can be one started with `Task.start_link`), monitor it and receive its error status (which may be Postgrex.Error).